bundles/mapping/camera-controls-3d/instance.js

```diff
@@ -0,0 +1,63 @@
+/*
+Oskari.app.playBundle(
+{
+  bundlename : 'camera-controls-3d'
+});
+*/
+Oskari.clazz.define('Oskari.mapping.cameracontrols3d.CameraControls3dBundleInstance',
```

Comment:
Consider extending the BasicBundle like here: https://github.com/oskariorg/oskari-frontend/blob/master/bundles/admin/admin-layereditor/instance.js#L5-L8

Reply:
Done